packages/fxa-auth-server/lib/senders/emails/templates/postRemoveRecoveryPhone/en.ftl

```diff
@@ -1,5 +1,6 @@
 postRemoveRecoveryPhone-subject = Recovery phone removed
 postRemoveRecoveryPhone-preview = Account protected by two-step authentication
 postRemoveRecoveryPhone-title = Recovery phone removed
-postRemoveRecoveryPhone-description = Your recovery phone has been removed from your two-step authentication settings. You still have your backup authentication codes available for use.
+postRemoveRecoveryPhone-description-v2 = Your recovery phone has been removed from your two-step authentication settings.
+postRemoveRecoveryPhone-description-extra = You can still use your backup authentication codes to sign in if you aren't able to use your authenticator app.
```

Comment:
```suggestion
postRemoveRecoveryPhone-description-extra = You can still use your backup authentication codes to sign in if you arenʼt able to use your authenticator app.
```

Reply:
I kind of forgot about this, seems like a good lint rule 🙃